lighthouse-core/test/gather/gatherers/dobetterweb/response-compression-test.js

```diff
@@ -191,27 +191,10 @@ describe('Optimized responses', () => {
       ],
     };
 
-    return responseCompression.afterPass(options, createNetworkRequests(traceData))
+    return responseCompression.afterPass(options, traceData)
       .then(artifact => {
         assert.equal(artifact.length, 1);
         assert.equal(artifact[0].resourceSize, 123);
       });
   });
-
-  // Change into SDK.networkRequest when examples are ready
-  function createNetworkRequests(traceData) {
-    traceData.networkRecords = traceData.networkRecords.map(record => {
-      record.url = record.url;
-      record.statusCode = record._statusCode;
-      record.mimeType = record.mimeType;
-      record.resourceSize = record.resourceSize;
-      record.transferSize = record.transferSize;
-      record.responseHeaders = record.responseHeaders;
-      record.requestId = record.requestId;
-
-      return record;
-    });
```

Comment:
this is handling suuuuper legacy stuff, from back when we used `SDK.networkRequest` for network request handling and had a split between leading-underscored internal properties and no-underscored external interface, so this was doubling up the values. Over time we migrated off using the underscores directly and then stopped using `SDK.networkRequest` altogether (almost three years ago - #5451), but we never got rid of this thing doing 85% nothing and 15% useless work :)

Anyways, `_statusCode` isn't a thing anymore except in this file, so 👋